neurovault/apps/statmaps/models.py

```diff
@@ -347,3 +351,33 @@ class Atlas(Image):
 
     class Meta:
         verbose_name_plural = "Atlases"
+
+
+class Similarity(models.Model):
```

Comment:
I think it's important because the universe of "kinds" of metrics and comparisons that we make is going to get large, and we need to keep it properly organized.  For example, a simple pearson could have a voxelwise brainmask version, a voxelwise pairwise deletion, some kind of regional version, etc. And we will most definitely be adding different metrics that are better suited for different image transformations and goals of the researcher.  It's important to have these separate from the actual Comparison data because at some point we will have a search page where the user is looking over his/her different options, and not only will we probably want to store more information about the metrics, but we definitely don't want to need to derive them from the monster table of comparisons themselves. For these reasons, I decided to model the two in this way.